tests/README.md

```diff
@@ -1,64 +1,80 @@
 
-SWMM-NRTestSuite
-================
+# SWMM Regression Testing
 
-Installation
-------------
+## Prerequisits
 
-Satisfying dependencies?
-nrtest swmm was develop on Windows using Python 2.7 64 bit.  
+Running SWMM's regression test suite `swmm-nrtestsuite` requires installation of the following software. 
+* git 
+* C compiler - MSVC, gcc, xcode
+* cmake
+* python 2.7 including setup tools 
+* swig
 
-Third party package dependencies:
-header_detail_footer
-nrtest 
-numpy 
 
-Project packages: 
-nrtest_swmm
-swmm-output
+## Step by Step Guide for Linux and MacOS
 
+The following are step by step instructions to compare the current SWMM OWA build against the SWMM 5.1.12 MSVC 32 bit benchmark. 
 
-pip can be used to install local packages. 
+1. Clone the swmm github repository. 
+```
+$ git clone --depth=50 --branch=feature-nrtest https://github.com/OpenWaterAnalytics/Stormwater-Management-Model.git
+```
 
-	$ pip install -r tools/requirements.txt
+2. Make repository root the current working directory
+```
+$ cd Stormwater-Management-Model
+```
 
+3. Install the required python packages. 
+```
+$ pip install --src buildprod/packages -r tools/requirements.txt 
+```
 
-Running Tests 
--------------
+4. Build swmm using cmake. 
+```
+$ cd buildprod
+$ cmake -DCMAKE_BUILD_TYPE=Release ..
+$ cmake --build . --config Release
+```
 
-working directory: 
-Change working directory to current location of 'swmm-nrtestsuite'
+5. Configure and run the regression tests: where <build id> - is the build identifier (i.e. swmm version number).
+```
+$ cd ..
+$ tools/gen-config.sh `pwd`/buildprod/bin > ./tests/apps/swmm-<build id>.json
+$ tools/run-nrtest.sh `pwd`/tests/swmm-nrtestsuite <build id>
+```
```

Comment:
Maybe we could add an example for an specific version, as it would make things more clear

Reply:
This is as straight forward as it needs to be.

Reply:
It can be better and that is the idea of having Pull Request reviews, to propose improvements. Having an actual example for an actual version is better than using placeholders. It is just 2 lines and it makes it much better

For example for version 5.x.y of SWMM you can do

```
$ tools/run-nrtest.sh `pwd`/tests/swmm-nrtestsuite 5.x.x
```